components/value-distribution/src/main/java/org/datacleaner/result/ValueCountingAnalyzerResult.java

```diff
@@ -61,7 +61,7 @@
 
     public Integer getUnexpectedValueCount();
 
-    public boolean hasAnnotatedRows(String value);
+    public Boolean hasAnnotatedRows(String value);
```

Comment:
Hmm that's very subtle ... Hard to guess even. I would rather just return false then and require the code to call this on the child results.